Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM hathitrust/feed_base:buster
+
+RUN apt-get install -y libtest-class-perl libswitch-perl libtest-spec-perl libtest-mockobject-perl
+
+WORKDIR /htapps/babel/imgsrv
+COPY . /htapps/babel/imgsrv
+
+RUN git config --global url."https://github.com/".insteadOf git@github.com:
+RUN git config --global url."https://".insteadOf git://
+
+RUN git submodule init
+RUN git submodule update
+RUN ln -s imgsrv/vendor/common-lib/lib ../mdp-lib
+RUN ln -s /htapps/babel /htapps/test.babel
+RUN cd ./sql; ln -s ../vendor/common-lib/lib/sql/*.sql .
```

Comment:
It seems like what you want here is for the stuff in common-lib to be visible outside the container, or rather, to be visible to the mariadb container when you first start it. If that's the purpose, I don't think this will actually work. I think it would be less brittle and simpler here to assume that the submodules have been set up correctly before building the docker image (à la https://github.com/hathitrust/feed/blob/master/.travis.yml#L9)

Reply:
The `git submodule update`? OK, I haven't done an iteration with a fresh checkout since this latest wiring to see if this isn't doing anything. 

The sql symlinks _do_ seem to work, though --- I end up with a working mysql setup.

Reply:
I'll try it in a clean environment to see how it's working.

Reply:
With a clean environment without the submodule not checked out on the host filesystem, this doesn't work -- it only attempts to load 100_ht_web_schema.sql because that's what's in the `sql` directory on the host filesystem. I think this will be (in part) obviated by HT-2670. Happy to discuss more why this is the case. For now, I still think getting the submodules set up outside of the context of building the Docker image is likely to be simpler.

Reply:
Did not realize how much was cached even doing `docker image rmi`. Chucked trying to do much in the Dockerfile in expectation of HT-2670.